fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1157,7 +1157,17 @@ ACTOR Future<Void> includeServers( Database cx, vector<AddressExclusion> servers
 					includeAll = true;
 				} else if (s.isWholeMachine()) {
 					// Eliminate both any ip-level exclusion (1.2.3.4) and any port-level exclusions (1.2.3.4:5)
-					tr.clear( KeyRangeRef( encodeExcludedServersKey(s), encodeExcludedServersKey(s) + char(':'+1) ) );
+					// Eliminate both any ip-level exclusion (1.2.3.4) and any
```

Comment:
It looks like the first part of this comment is duplicated

Reply:
Fixed